README.md

```diff
@@ -5,14 +5,14 @@ database.
 
 You can build and run the project with [stack](http://haskellstack.org/), e.g.:
 
-``` bash
+```shell
```

Comment:
Why did you change this from `bash` to `shell`?

Reply:
[Github Help](https://help.github.com/articles/creating-and-highlighting-code-blocks/) points to this [definition](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L4187-L4195) of possible languages in markdown code blocks. `shell`, `bash`, `zsh` and others are aliases. The change does effectively nothing. I'd prefer the keyword `shell` because it's agnostic and the named commands work in any shell, not only the bash shell. But all in all this is absolutely a minor change.

Reply:
Thanks for the explanation, makes sense.